blocks/library/subhead/index.js

```diff
@@ -61,13 +61,6 @@ export const settings = {
 		const { content, placeholder } = attributes;
 
 		return [
```

Comment:
We can get rid of the array return here.

Reply:
@mtias Of course, I removed the array.